database/mongodb/new.go

```diff
@@ -5,11 +5,13 @@ import (
 	"time"
 
 	"github.com/globalsign/mgo"
+	"github.com/ohsu-comp-bio/funnel/compute/scheduler"
 	"github.com/ohsu-comp-bio/funnel/config"
 )
 
 // MongoDB provides an MongoDB database server backend.
 type MongoDB struct {
+	scheduler.UnimplementedSchedulerServiceServer
```

Comment:
Why is this needed?

Reply:
You could get rid of this if you use `--go-grpc_opt=paths=source_relative,require_unimplemented_servers=false`